rules-reviewed/openshift/session.windup.xml

```diff
@@ -25,63 +25,77 @@
               </xmlfile>
             </when>
             <perform>
-                <hint title="HTTP Session replication" effort="5" category-id="openshift-mandatory">
+                <hint title="HTTP Session replication" effort="3" category-id="cloud-mandatory">
                     <message>
                     <![CDATA[
-                       The `web.xml` deployment descriptor for a Web application archive contains the `<distributable>` subelement.  
-                       To configure EAP clustering in OpenShift, please follow the instruction in the following links:
+                       Session replication ensures that client sessions are not disrupted by failovers of nodes in a cluster.  
+                       Each node in the cluster shares information about ongoing sessions and can take over sessions if a node disappears.  
+                       To enable clustering of a web application, you must configure the application to be distributable [1][6]:  
+                       the `<distributable/>` element inside the `<web-app>` tag of your application’s `web.xml` descriptor file indicates that application's sessions have to be replicated.  
+                       
+                       In a cloud environment it has to be considered that the data in the memory of a running container can be wiped out by a restart (triggered by code deploy, config change, or the execution environment relocating the process to a different physical location)[2].  
+                       
+                       Different approaches can be followed:
+                       
```

Comment:
Would add also an option:

* disable HTTP session clustering and accept its implications